engine/swarm/index.md

```diff
@@ -4,17 +4,16 @@ keywords: docker, container, cluster, swarm
 title: Swarm mode overview
 ---
 
-To use Docker Engine in swarm mode, install the Docker Engine `v1.12.0` or
-later from the [Docker releases GitHub
-repository](https://github.com/moby/moby/releases). Alternatively, install
-the latest Docker for Mac or Docker for Windows Beta.
+To use Docker in swarm mode, install Docker `1.12.0` or later. Alternatively,
+install the latest Docker for Mac or Docker for Windows. Install instructions
+for all platforms are [here](https://docs.docker.com/engine/installation/). The
+most recent version is Docker 17.06.
```

Comment:
I'd be cautious putting "latest version" in this document, as it easily gets outdated. Given that Docker versions older than 1.12 are no longer supported, perhaps we can get rid of this section.